src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -654,10 +632,9 @@ def __call__(self):
 
             if funcName == "repack":
                 self.handleRepackSettings()
-            elif funcName == "merge":
-                self.handleMergeSettings()
-            elif funcName == "alcaHarvesting":
-                self.handleAlcaHarvestSettings()
+
+            if funcName in [ "repack", "merge", "alcaharvest" ]:
```

Comment:
No need for the trailing spaces inside the list (Dirk, try running pylint before pushing these commits upstream).
Second comment is a typo in your function name, should be H upper case like `alcaHarvesting`